tests/Functional/tests/render/code-textrole-no-xss/code-textrole-no-xss.rst

```diff
@@ -0,0 +1,5 @@
+``<script>alert('hello!')</script>``
+
+`<script>alert('hello!')</script>`
+
+:code:`$this->execute(){}`
```

Comment:
Right now ```
:code:`<script>alert('hello!')</script>`
``` fails due to the last part being interpreted as URL target. I will provide a follow up for this

Reply:
This problem is unrelated and I did not want to mix it in here